addons/area_markers/functions/fnc_configure.sqf

```diff
@@ -0,0 +1,193 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Opens a menu to configure the properties of the given area marker.
+ *
+ * Arguments:
+ * 0: Marker <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ["marker_0"] call zen_area_markers_fnc_configure
+ *
+ * Public: No
+ */
+
+#define DEGREE_SYMBOL (toString [176])
```

Comment:
Isn't this defined for the second time? I think it was defined in the dynamic dialog slider as well. Perhaps extract it to common?